src/main/java/org/cookieandkakao/babting/domain/food/controller/FoodPreferenceController.java

```diff
@@ -38,22 +41,26 @@ public FoodPreferenceController(PreferenceFoodService preferenceFoodService,
 
     // 선호/비선호 음식 조회
     @GetMapping("/{type}")
-    public ResponseEntity<?> getFoodPreferences(
+    @Operation(summary = "선호/비선호 음식 조회", description = "내 선호/비선호 음식을 가져옵니다.")
+    @ApiResponse(responseCode = "200", description = "선호/비선호 음식 조회 성공")
+    public ResponseEntity<ApiResponseBody.SuccessBody<List<FoodPreferenceGetResponse>>> getFoodPreferences(
             @PathVariable String type,
             @LoginMemberId Long memberId
```

Comment:
```suggestion
            @Parameter(hidden = true) @LoginMemberId Long memberId,
```
memberId는 swagger 문서에 안보이게 설정해주면 좋을 것 같아요

Reply:
넵!!! 추가하겠습니다